tests/loadtests/vkloadtests/VulkanLoadTestSample.cpp

```diff
@@ -75,7 +78,8 @@ namespace VMA_CALLBACKS
     {
         uint64_t allocId = mt64();
         VmaAllocationCreateInfo pCreateInfo = {};
-        if (allocInfo->memoryTypeIndex == 8)
+        if ((cachedDevMemProps.memoryTypes[allocInfo->memoryTypeIndex].propertyFlags & VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT) ||
+            (cachedDevMemProps.memoryTypes[allocInfo->memoryTypeIndex].propertyFlags & VK_MEMORY_PROPERTY_HOST_COHERENT_BIT))
```

Comment:
Leave these 2 alone as splitting the &s' operands will make it too hard to follow the logic.